packages/atomic-hosted-page/package.json

```diff
@@ -21,9 +21,8 @@
   ],
   "scripts": {
     "clean": "rimraf -rf dist/*",
-    "build": "npm run build:stencil && npm run validate:definitions",
-    "build:stencil": "node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build",
-    "dev": "npm run build:stencil -- --dev --watch --serve",
+    "build": "nx build",
+    "dev": "node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build --dev --watch --serve",
```

Comment:
Any task whose command was moved to `project.json`, I'm removing from the scripts here.

It would be nice, eventually, to have only useful tasks here.